vevestaX/vevesta.py

```diff
@@ -791,9 +790,18 @@ def __EDAForPandas(self, fileName):
         plt.suptitle('Feature Histogram',fontsize=20)
         plt.savefig(os.path.join(directoryToDumpData, FeatureHistogramImageFile), bbox_inches='tight', dpi=100)
         plt.close()
+        pdffile="EDA.pdf"
+        images=[ValueImageFile,ValueRatioImageFile,NumericalFeatureDistributionImageFile,NonNumericFeaturesImgFile,FeatureHistogramImageFile,OutliersImageFile]
+        a4inpt = (img2pdf.mm_to_pt(210),img2pdf.mm_to_pt(297))
```

Comment:
use better variable name for a4inpt and layout_fun

Reply:
changed the variables